### DISCUSSION TITLE
Custom domain is not applied to all repositories.

### DISCUSSION THREAD

**Discussion by minw0525** [community/community#68305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've applied a custom domain to one of my GitHub Pages hosted websites. 
According to the official documentation ( https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages ), by applying a custom domain, it will be applied to all other projects I own. 
However, the custom domain is not applied to the URLs of my other repository pages, and they are deployed to the URL of username.github.io/reponame.
Is there any way to apply one custom domain to all repository pages?

**Comment by edrf12** [comment]
Did you manage to solve this? I have the same issue and have not figured it out

**Reply by minw0525** [reply]
Initially, I applied a custom domain to my project site at {username}.github.io/{repository} and the domain was not applied to the entire repository.
When I created a user site repository named {username}.github.io and applied the custom domain there, the custom domain was applied to the entire repository. 

https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites
